code/modules/robotics/bot/guardbot.dm

```diff
@@ -364,7 +364,7 @@
 	mail
 		name = "Mailbuddy"
 		desc = "The PR-6PS Mailbuddy is a postal delivery ace.  This may seem like an extremely specialized robot application, but that's just because it is exactly that."
-		icon = 'icons/obj/mailbud.dmi'
+		icon = 'icons/obj//bots/mailbud.dmi'
```

Comment:
no.
```suggestion
		icon = 'icons/obj/bots/mailbud.dmi'
```

Reply:
Oops- oddly enough it still worked when i tested it